art/step/step.py

```diff
@@ -36,3 +35,6 @@ def _get_saved_state(self) -> Dict[str, str]:
 
     def get_saved_state(self) -> Dict[str, str]:
         return {}
+
+    def get_step_id(self) -> str:
```

Comment:
I do not think that steps should have different names based on which datamodule or model was used. Maybe for different model we should just do it in a different folder